include/configuration.h

```diff
@@ -256,6 +256,9 @@ struct Config {
 
     /* Disable gaps if there is only one container on the workspace */
     bool smart_gaps;
+
+    /* Disable list of windows in stacked or tabbed mode */
```

Comment:
It's not so much a »list of windows«. It's the titlebar of the tabbed or stacked container.